### PR TITLE
add modifier onlyOwner in Ownable

### PR DESCRIPTION
**Pull request by kumaryash90**

### CODE REVIEW

File: contracts/feature/Ownable.sol [hunk 1]
```diff
@@ -11,7 +11,20 @@ import "./interface/IOwnable.sol";
...
+    /// @dev Reverts if caller is not the owner.
+    modifier onlyOwner() {
+        if (msg.sender != _owner) {
+            revert Ownable__NotAuthorized();
+        }
+        _;
+    }
+
+    /// @dev Returns the owner of the contract.
+    function owner() public view returns (address) {
```
Comment (joaquim-verges) [inline contracts/feature/Ownable.sol:25]: if i remember correctly, `owner` is a standard that things like Opensea fetches, i think this should be fine, but we should make sure its still compatible? @jakeloo @nkrishang do you know?

File: contracts/feature/Ownable.sol [hunk 2]
```diff
@@ -11,7 +11,20 @@ import "./interface/IOwnable.sol";
 
 abstract contract Ownable is IOwnable {
     /// @dev Owner of the contract (purpose: OpenSea compatibility)
-    address public override owner;
+    address private _owner;
```
Comment (joaquim-verges) [inline contracts/feature/Ownable.sol:14]: i'm expecting to see a change in the interface as well but im not seeing it in this PR?

Comment (kumaryash90) [reply contracts/feature/Ownable.sol:14]: Interface should remain the same. It has a function: `owner()`, which was previously overridden by the owner variable which was public.

Comment (joaquim-verges) [reply contracts/feature/Ownable.sol:14]: ah yes you're right

**Review by joaquim-verges — APPROVED** [review]
ship it

**Pull request merged**